integration-tests/tests/src/tests/sync/app.ts

```diff
@@ -16,21 +16,23 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-import { BSON } from "realm";
+import { BSON, ObjectSchema, Realm, App, Credentials, SyncConfiguration } from "realm";
 import { expect } from "chai";
 import { importAppBefore } from "../../hooks";
 import { generatePartition } from "../../utils/generators";
 import { getUrls } from "../../utils/import-app";
 import { select } from "../../utils/select";
 
-const TestObjectSchema: Realm.ObjectSchema = {
+const TestObjectSchema: ObjectSchema = {
```

Comment:
I understand this is tempting, but we've talked about avoiding refactoring these tests until they all pass 🤞

It's not a blocker for me, but just something to keep in mind for any future PRs.